### ISSUE TITLE
implement GCN layer

### ISSUE THREAD

**Issue by 0h-n0** [clinfo/kGCN_torch#2]

**Comment by 0h-n0** [comment]
@kojima-r  @tarokiritani
In the previous meeting, we talked about pytorch_geometric which is the most popular graph neural network library in pytorch. I'm wondering whether kGCN_torch is based on pytorch_geometric or not.

**Comment by kojima-r** [comment]
You can use pytorch_geometric, while you have to be careful not to mistake the handling of "channel" in kGCN. The standard format in kGCN is a multi-edge graph although pytorch_geometric's standard might be a single-edge graph.

**Comment by 0h-n0** [comment]
Thanks. I will carefully implement GCN layer.  So, you mean that original kGCN supports multi state graph? Each states are corresponding to each channels, right?

**Comment by kojima-r** [comment]
Yes. 
""" psedo code of GraphConv for multi-edge graph
adjs = channel x node x node
X= batch x node x feature(input)
W=channel x feature(input) x feature(output)
for i in batch:
>for k in channel:
>>output[i,k]=A[k] * X[i,k] * W[k]

sum(output,axis=1)

"""

This can be realized by single-edge GraohConv (SingleGraphConv) as follows:


"""
out=[]
for k in channel
>out.append(SingleConv(adjs[k],X))

sum(out)

"""

**Comment by 0h-n0** [comment]
I implemented the layer.

https://github.com/clinfo/kGCN_torch/blob/bbbae28832709623058cbfb34e93fdff1ea38b3f/kgcn_torch/nn/multiedge_gcn.py#L12-L39
